pom.xml

```diff
@@ -1196,6 +1198,8 @@
             </build>
 
             <modules>
+                <module>dbunit</module>
+
```

Comment:
You'll be transplanting these to another pom file - watch out for extraneous spacing when you do.